### DISCUSSION TITLE
Prettier plugin

### DISCUSSION THREAD

**Discussion by jakubmazanec** [geldata/gel#4242] [Q&A] [answered]
Is there a Prettier plugin for formatting ESDL files? Or plans for one?

**Comment by colinhacks** [comment, marked as answer]
Not yet, though we're planning to build a full [LSP](https://en.wikipedia.org/wiki/Language_Server_Protocol) integration for EdgeQL. No timeline yet I'm afraid.

**Reply by jakubmazanec** [reply]
Ok, thanks for the reply.
